addons/workspace-dots/userscript.js

```diff
@@ -0,0 +1,58 @@
+export default async function ({ addon, console }) {
+  const ScratchBlocks = await addon.tab.traps.getBlockly();
+
+  // https://github.com/scratchfoundation/scratch-blocks/blob/develop/core/grid.js#L136
+  const oldUpdate = ScratchBlocks.Grid.prototype.update;
+  ScratchBlocks.Grid.prototype.update = function (scale) {
+    const spacingMultiplier = addon.settings.get("spacingMultiplier");
+    if (!addon.self.disabled) this.spacing_ /= spacingMultiplier;
+    oldUpdate.call(this, scale);
+    if (!addon.self.disabled) this.spacing_ *= spacingMultiplier;
```

Comment:
Storing the original spacing is better, especially if the calculation becomes more complex - see above.

Reply:
I will do this